onnxruntime/test/python/onnxruntime_test_python.cc

```diff
@@ -0,0 +1,1022 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
the test is showing up as a new file?
why does it have a .cc extension ? did you rename it?

Reply:
it's fixed